content/al/0018-matriz-mudanca-de-coordenadas.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Matriz de mudança de coordenadas
+description: >-
+  Como obter uma matriz de mudança de coordenadas
+path: /al/matriz-mudanca-de-coordenadas
+type: content
+---
+
+# Matriz de mudança de coordenadas
+
+```toc
+
+```
+
+## Matriz de mudança de coordenadas
+
+:::tip[Definição]
+
+Sejam $B_1=\{v_1,...,v_n\}$ e $B_2=\{w_1,...,w_n\}$, duas bases ordenadas de um espaço vetorial $V$,
+a matriz $S_{B_1 \to B_2} = (s_{ij})_{n\times n}$, cujas colunas são as coordenadas dos vetores de $B_1$ escritos na base $B_2$,
+chama-se matriz de mudança de coordenadas de $B_1$ para $B_2$.
+
+:::
+
+:::tip[Teoremas]
```

Comment:
I don't think this should be a container at all, just regular text

Reply:
Agree